vignettes/tidytranscriptomics.Rmd

```diff
@@ -375,19 +390,15 @@ An alternative to MDS for examining relationships between samples is using hiera
 counts_scal_MDS %>% 
 	
 	# extract 500 most variable genes
-	keep_variable(
-	              .sample = sample, 
-	              .transcript = transcript, 
-	              .abundance = count_scaled, 
-	              top = 500) %>%
+	keep_variable( .abundance = count_scaled, top = 500) %>%
 		
 	# create heatmap
 	heatmap(
 	      .column = sample,
 	      .row = transcript,
 	      .value = count_scaled,
 	      annotation = c(dex, cell),
-	      transform = log1p
+	      transform = function(x) x %>% log1p %>% tidyHeatmap:::scale_robust()
```

Comment:
this is a failure here:

```Quitting from lines 390-403 (tidytranscriptomics.Rmd) 
Error: processing vignette 'tidytranscriptomics.Rmd' failed with diagnostics:
object 'scale_robust' not found
--- failed re-building ‘tidytranscriptomics.Rmd’

is scale_robust only available in tidybulk dev? we're supposed to be able to use bioc devel so is there a way to get it into that?

Reply:
I will make it exported from tidyHeatmap and update CRAN. Strange that gives error to you. Aa maybe because you have the cran version of course. I will update CRAN don't worry

Reply:
In the meanwhile could you please update tidyHeatmap from github?

Reply:
I could but that won't stop the failure, because Github actions is automatically installing the packages from CRAN and Bioc devel, you can see the details below if you haven't already
<img width="925" alt="Screen Shot 2020-05-30 at 7 17 50 pm" src="https://user-images.githubusercontent.com/5591821/83324736-b7340080-a2aa-11ea-805c-40cf27ca3043.png">

<img width="1419" alt="Screen Shot 2020-05-30 at 7 17 21 pm" src="https://user-images.githubusercontent.com/5591821/83324754-c6b34980-a2aa-11ea-964d-f3d52a20ad8d.png">

If it fails it means the changes don't get pushed to our website that the Github actions are automatically building here https://stemangiola.github.io/bioc_2020_tidytranscriptomics/articles/tidytranscriptomics.html

So if the change will get to CRAN soon we could wait otherwise if it will take a while (more than a day?) it's probably better to remove that scale_robust so we can add the other changes, then add the scale_robust when it's in CRAN

Reply:
I see. The CRAN tidyHeatmap it does scale automatically. I see we were seing to different plots because you have the CRAN version. I as seing it not scaled and so very ugly.

In this case discart my change!

Reply:
ok will do 😄